doc/releases/changelog-0.33.1.md

```diff
@@ -14,6 +14,21 @@
 
```

Comment:
Will the categories above be deleted before release?

Reply:
certainly!